File: Javascript/Javascript Fundamentals/OperatorsAndExpressions/third-digit.js
function solve(args) {
	var a = +args[0] + "";

	var digit = a[a.length - 3];

	if (digit === "7") {
		console.log("true");
	}
	else if(a.length < 3){
		console.log("false 0");
	}	
	else{
		console.log("false " + digit);
	}
}

File: C#/C# Fundamentals/OperatorsAndExpressions/ThirdDigit/ThirdDigit.cs
﻿/* 
   Write a program that reads an integer N from the console and prints true if the third digit of the N is 7, 
   or "false THIRD_DIGIT", where you should print the third digits of N.

   The counting is done from right to left, meaning 123456's third digit is 4.
*/

using System;

class Digit
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        number = (number / 100) % 10;

        if (number == 7)
        {
            Console.WriteLine("true");
        }
        else
        {
            Console.WriteLine("false {0}", number);
        }
    }
}